docs/support-tasks.md

```diff
@@ -125,26 +125,24 @@ The date should be in ISO8601 format, for example 2020-01-01.
 To see a simple count of the number of subscribers for a given list:
 
 ```bash
-bundle exec rake 'report:csv_subscriber_lists[<url>]'
+bundle exec rake 'report:csv_subscriber_lists[<path>,<active_on_date (optional)>]'
 ```
-This will always report on the number of active subscriptions for a given url, as of the end of today.
+This will always report on the number of active subscriptions for a given path, as of the end of today.
```

Comment:
I think this line needs a tweak in light of adding the data param in the snippet.